### ISSUE TITLE
PWIM Model

### ISSUE THREAD

**Issue by BruceLee66** [lanwuwei/SPM_toolkit#26]
I used the same sentence pair into the PWIM model, but each time the result is different, why is this?

**Comment by lanwuwei** [comment]
random seed? is the mapping between word and embedding index consistent?

**Comment by BruceLee66** [comment]
> random seed? is the mapping between word and embedding index consistent?

I have trained the model by myself and saved the parameter.
    pretrained_dict=torch.load('new_model_static.pkl')
    a='word_embedding.weight'
    b='copied_word_embedding.weight'
    pretrained_dict.pop(a)
    pretrained_dict.pop(b)
    model_dict=model.state_dict()
    model_dict.update(pretrained_dict)
    model.load_state_dict(model_dict)
    model.eval()

**Comment by BruceLee66** [comment]
I know the truth,some words do not exit.

**Comment by lanwuwei** [comment]
good!
